DEVELOPMENT.md

```diff
@@ -26,6 +26,25 @@ To run end-to-end tests you will also need:
 
 Installation instructions for these tools differ heavily based on your operating system and aren't covered here.
 
+### CouchDB on Docker
+
+We recommend using Docker to install and use CouchDB. This ensures you are getting a compatible version and are not relying on OS packages that may be more or less up to date than needed. 
+
+After installing docker, you can create a docker container like so:
+
+```sh
+docker run -d -p 4984:5984 -p 4986:5986 --name medic-couchdb -v /home/scdf/dockers/e2e-couchdb/opt/couchdb/data:/opt/couchdb/data apache/couchdb:2
```

Comment:
Consider replacing your local dir with say `<path>` so it's easier to find and replace. eg: 
```suggestion
docker run -d -p 4984:5984 -p 4986:5986 --name medic-couchdb -v <path>:/opt/couchdb/data apache/couchdb:2
```